### ISSUE TITLE
Minor visual bug in counting on Dashboard chart

### ISSUE THREAD

**Issue by Bangalisch** [btcpayserver/btcpayserver#3768]
Yo yo. 

just launched a new store in my dev env. Ran a couple of cheats to up my stores balance. 
And there's when I noticed something odd. Attached is visual of what happens. 

So I went from 0 bitcoin to a whole lot, but due to the nature of the cheat, it places it over a time span and not all just today. 
When now looking in the weekly the counting in the left is bugged. Whenever I switch between week month or year its goes back to being oke. tho, switching menus (lets say goto settings and back to dashboard) brings backup the wrong counting. 

Can't tell if this is something related to testing env :) 
Cheers

![ezgif com-gif-maker](https://user-images.githubusercontent.com/102448109/169655699-33260dfe-3fbe-40d7-835f-68bcb257062b.gif)

<img width="682" alt="Screenshot 2022-05-21 at 15 51 40" src="https://user-images.githubusercontent.com/102448109/169655712-e9ca6824-b562-4e25-b1e5-4790538eae31.png">

**Comment by dennisreimann** [comment]
I think this got fixed with btcpayserver/btcpayserver#3702 - can you check which version you are running? The fix was released with [1.5.2](https://github.com/btcpayserver/btcpayserver/releases/tag/v1.5.2).

**Comment by NicolasDorier** [comment]
I can't repro on master

**Comment by NicolasDorier** [comment]
@Bangalisch closing this, I release 1.5.3, I think we fixed it. If not the case, please reopen the issue.

**Comment by Bangalisch** [comment]
> I think this got fixed with #3702 - can you check which version you are running? The fix was released with [1.5.2](https://github.com/btcpayserver/btcpayserver/releases/tag/v1.5.2).

1.5.2.0 as it shows right bottom corner. 
No crazy PR's, just master branch. 
The only thing different from the previous time I noticed, this regrets is build on mac and is looked at in Safari instead of Firefox. I would be surprised if that's to do with it, but Ill update to the 1.5.3 branch and see if it still exists. 
Its not so much the breaks on the counting, its literally not counting right.

<img width="1440" alt="Screenshot 2022-05-25 at 14 59 31" src="https://user-images.githubusercontent.com/102448109/170267731-2b9aca31-7f28-40dd-a057-701d8570cc19.png">

**Comment by NicolasDorier** [comment]
yeah that's strange. You can reliably reproduce it on regtest with safari? Maybe it's a browser thing.

**Comment by Bangalisch** [comment]
> yeah that's strange. You can reliably reproduce it on regtest with safari? Maybe it's a browser thing.

I'll go through the following steps : 
Delete postgres and restart with a fresh one on the same branch. If same result, update to latest branch. 
Delete Postgres again and redo it on new branch. See if results still same. 
If results the same, switch browsers. redo it all. :)
